docs/testing/continuous-integration.md

```diff
@@ -52,16 +49,40 @@ uses those SSH keys for authentication.
 
 The main CircleCI configuration file is
 [./circleci/config.yml](https://github.com/zulip/zulip/blob/master/.circleci/config.yml).
-We run multiple jobs during a CircleCI build to test different
-supported platforms. They are currently:
+We run multiple jobs during a CircleCI build to test both development and production environment
```

Comment:
I didn't like this framing around "development and production environment"s -- the goal of testing is to ensure things work in production; the development environment exists almost entirely in service of that purpose.  The right framing is that we're running the test suites we can run in the development environment plus some end-to-end tests.